src/main/java/at/hannibal2/skyhanni/utils/ComputerTimeOffset.kt

```diff
@@ -58,15 +59,16 @@ object ComputerTimeOffset {
         timeInfo.computeDetails()
         timeInfo.offset.milliseconds
     } catch (e: Exception) {
-        e.printStackTrace()
+        if (LorenzUtils.inSkyBlock && config) ErrorManager.logErrorWithData(e, "Failed to get NTP offset")
+        else e.printStackTrace()
         null
     }
 
-    var lastSystemTime = System.currentTimeMillis()
-    var lastDetectedOffset = 1.seconds
+    private var lastSystemTime = System.currentTimeMillis()
+    private var lastDetectedOffset = 1.seconds
 
     private fun startMonitoringTimeChanges() {
-        thread(start = true) {
```

Comment:
why removing the start value? this will again never run

Reply:
![image](https://github.com/user-attachments/assets/5f16aa09-1441-41a2-8d1b-0de8ba7d2868)
start true if the default value